src/loweringc.jl

```diff
@@ -0,0 +1,246 @@
+##
+# Methods to compute the Jacobi operator for (t,a,b,c-1) based on knowledge of (t,a,b,c)
+##
+
+# directly evaluate α[1] via hypergeometric functions, making use of the fact that Q_1 == A[1]x + B[1]
+function initialαc_gen(t, a, b, c)
+    A,B,_ = recurrencecoefficients(SemiclassicalJacobi(t,a,b,c))
+    return -(A[1]*t^(c-1)*gamma(2+a)*gamma(1+b)/gamma(3+a+b)*_₂F₁general2(a+2,-c+1,a+3+b,1/t)/(t^(c-1)*gamma(1+a)gamma(1+b)/gamma(2+a+b)*_₂F₁general2(1+a,-c+1,2+a+b,1/t))+B[1])
```

Comment:
It's the same as https://github.com/JuliaApproximation/SemiclassicalOrthogonalPolynomials.jl/blob/bd1af796b0e455aa2ef5bdd60323bef4300e97fb/src/SemiclassicalOrthogonalPolynomials.jl#L47. It may be, a beta function but I now just changed it to refer to that function instead.